bee-message/tests/ed25519_signature.rs

```diff
@@ -6,7 +6,7 @@ use bee_message::prelude::*;
 
 use core::convert::TryInto;
 
-const ED25519_PULIC_KEY: &str = "1DA5DDD11BA3F961ACAB68FAFEE3177D039875EAA94AC5FDBFF8B53F0C50BFB9";
+const ED25519_PUBLIC_KEY: &str = "1DA5DDD11BA3F961ACAB68FAFEE3177D039875EAA94AC5FDBFF8B53F0C50BFB9";
```

Comment:
I know it was already there but could you make it lowercase for consistency ?

Reply:
Sure, didn't notice that!